apps/webapp/src/state/swap.ts

```diff
@@ -55,51 +54,32 @@ export const createSwapSlice = (): SliceCreator<SwapSlice> => (set, get) => {
         state.swap.txInProgress = true;
       });
 
-      const { dismiss, update } = toastFn(buildingTxToast());
+      const { update } = toastFn(buildingTxToast());
 
       try {
         // build swap
-        const swapPlan = await plan(await assembleSwapRequest(get().swap));
+        const swapReq = await assembleSwapRequest(get().swap);
+        const swapPlan = await plan(swapReq);
         const swapTx = await authWitnessBuild(update, { transactionPlan: swapPlan });
 
         // begin broadcast
-        const swapBroadcast = broadcast(update, {
-          awaitDetection: true,
-          transaction: swapTx,
-        });
-
-        // a swap claim must be broadcast after any successful swap, to claim
-        // the output of the swap.  Swap claims don't need authenticationData,
-        // so use `witnessAndBuild`
-        // @see https://protocol.penumbra.zone/main/zswap/swap.html#claiming-swap-outputs
-
-        // start to build swap claim
-        const swapCommitment = getSwapCommitment(swapTx);
-
-        const swapClaimPlan = await plan(assembleSwapClaimRequest(swapCommitment));
-        const swapClaimTx = await witnessBuild(update, { transactionPlan: swapClaimPlan });
+        await broadcast(update, { awaitDetection: true, transaction: swapTx });
 
-        // after swap broadcast success, emit the claim
-        await swapBroadcast;
```

Comment:
In order to plan a swap claim, wasm needs access to the swap record in our db. So this will throw.